packages/react-progress-bar/lib/react-progress-bar.js

```diff
@@ -31,12 +36,12 @@ export const ProgressBar = ({
 	);
 
 	const loaderText = hasLabel && (
-		<span className="sui-progress-text">{'' === value ? 0 : value}%</span>
+		<span className="sui-progress-text">{value}%</span>
```

Comment:
@creador-dev, notice in the screenshot that the second example uses the “hello” string as now property value. Instead of returning zero value, it returns 100% for the progress bar and “hello%” for the progress label. That is something we need to correct.

<img width="1287" alt="Screen Shot 2022-05-21 at 10 46 39" src="https://user-images.githubusercontent.com/2328848/169659205-cabf8b58-6869-4ed5-b3a7-cb69fb3b69ab.png">

Reply:
@creador-dev, I left you my comments/review on the JIRA task.